### ISSUE TITLE
[Feature Request] Random comic button

### ISSUE THREAD

**Issue by vguttmann** [zjn0505/xkcd-Android#23]
A "display a random comic" button would be nice, since the random function of the comic date is unintuitive, and the date is hard to reach with the thumb on a smartphone.

**Comment by AlexanderPosch** [comment]
While somewhat intuitive, I don't mind the position of the date. Although I would like the option for the phone not to vibrate when clicking it.

**Comment by hp27596** [comment]
I agree that pressing the date to get random comics are unintuitive. I was going to write a request for the random comic feature until I read this issue and realize I could do that. A separate random button in the middle would be nice and logical.
